### PR TITLE
Filled in BMS latching in IMD latching section

### PR DESCRIPTION
**Pull request by aehoppe**
Just filled in our intentions with BMS latching.

### CODE REVIEW

File: esf_main.tex [hunk 1]
```diff
@@ -388,23 +388,23 @@ \subsubsection{Description/circuitry}
...
+The IMD latch will be connected to the acummulator interface board board via a 4-pin Molex LLC Ultrafit 1722861104 with 20 AWG wire, and the IMD connector using a  6-pin TE Connectivity MICRO MATCH 338068-6 connector with 28 AWG ribbon cable. The 28 AWG ribbon cable has a cross sectional area of .08mm$^2$, and is rated for 105 \textdegree C, 300 V, and 1.4 amps. This is provides a considerable safety margin. The ribbon cable carries signals from the IMD. 18 AWG wire is used for the shutdown circuit and to supply power and ground to the IMD latch. The 18 AWG wire has a cross sectional area of .83mm$^2$, and is rated for 16 amps for chassis wiring and 2.3 amps for power transmision. The 18 wire will be rated for at least 90 \textdegree C and 300~V, as required by the rules for wiring in the accumulator (see EV3.3.8 and EV4.5.4).
```
Comment (byronwasti) [inline esf_main.tex]: Small typo.

> This ~~is~~ provides a considerable safety margin.

Also I believe the AWG is equivalent to cross-sectional area, so it is redundant to include both.

File: esf_main.tex [hunk 2]
```diff
@@ -388,23 +388,23 @@ \subsubsection{Description/circuitry}
...
 
+Similarly to the IMD latch, the BMS latch will be housed in the accumulator, and thus all for the shutdown circuit to and from it will be conneted with 18AWG wire rated to over 300V and 90 \textdegree C in order to handle the maximum of 8A inrush current to the AIRs and maintain compliance with accumulator wiring regulations. PCB traces will also be appropriately sized to handle 8A peak current.
```
Comment (byronwasti) [inline esf_main.tex]: > and thus all for the shutdown circuit to

Not really sure what you are going for.

> ~~conneted~~ connected

**Review by byronwasti — CHANGES_REQUESTED** [review]
Small typos.

**Comment by aehoppe** [comment]
I fixed some typos

**Pull request merged**